2018/g2018_vr.csv

```diff
@@ -37,19 +37,19 @@ ND,,,,,Voters do not register in North Dakota
 OH,2018-10-07,,,,
 OK,2018-10-12,2018-10-12,,,
 OR,2018-10-16,2018-10-16,,,
-PA,2018-10-07,2018-10-07,,,
+PA,2018-10-09,2018-10-07,,,
 RI,2018-10-07,2018-10-07,,,
 SC,2018-10-07,2018-10-07,,,
 SD,2018-10-22,,,,
-TN,2018-10-07,2018-10-07,,,
+TN,2018-10-09,2018-10-07,,,
 TX,2018-10-07,,,,
 UT,2018-10-30,2018-10-30,,,
-VT,,,,,"Voters in Vermont can register to vote on any day, including Election Day"
+VT,2018-11-06,,,,"Voters in Vermont can register to vote on any day, including Election Day"
```

Comment:
See the [other comment](https://github.com/democrats/election-calendar/pull/36/files#r170463780) and #10